### PR TITLE
Rename Port/Route Web Classes, and Update Kubernetes yaml

### PR DESCRIPTION
**Pull request by kevin-zhonghao**
This PR proposes the following change:
* Rename web classes of port and route to avoid duplicate with schema classes
* Add new Kubernetes yaml files of node manager and port manager

### CODE REVIEW

File: services/port_manager/src/main/java/com/futurewei/alcor/portmanager/controller/PortController.java [hunk 1]
```diff
@@ -36,16 +36,16 @@
      * If any exception occurs in the added process, we need to roll back
      * the resource allocated from each micro-service.
      * @param projectId Project the port belongs to
-     * @param portStateJson Port configuration
+     * @param portWebJson Port configuration
```
Comment (xieus) [inline services/port_manager/src/main/java/com/futurewei/alcor/portmanager/controller/PortController.java:39]: Thanks for updating the comment as well 👍

File: services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java [hunk 2]
```diff
@@ -50,7 +50,7 @@
             value = {"/vpcs/{vpcId}/routes/{routeId}"})
     public RouteWebJson getRuleByVpcId(@PathVariable String vpcId, @PathVariable String routeId) throws Exception {
 
-        RouteWebObject routeState = null;
+        RouteEntity routeState = null;
```
Comment (xieus) [inline services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java:53]: rename the variable name to "route"

File: services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/RouteDatabaseServiceImpl.java [hunk 3]
```diff
@@ -35,7 +35,7 @@ public Map getAllRoutes() throws CacheException {
     }
 
     @Override
-    public void addRoute(RouteWebObject routeState) throws DatabasePersistenceException {
+    public void addRoute(RouteEntity routeState) throws DatabasePersistenceException {
```
Comment (xieus) [inline services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/RouteDatabaseServiceImpl.java:38]: maybe rename the variable name to "route"

File: services/route_manager/src/main/java/com/futurewei/alcor/route/service/RouteDatabaseService.java [hunk 4]
```diff
@@ -4,15 +4,15 @@
...
 
 import java.util.Map;
 
 public interface RouteDatabaseService {
 
-    public RouteWebObject getByRouteId (String routeId) throws ResourceNotFoundException, ResourcePersistenceException;
+    public RouteEntity getByRouteId (String routeId) throws ResourceNotFoundException, ResourcePersistenceException;
     public Map getAllRoutes () throws CacheException;
-    public void addRoute (RouteWebObject routeState) throws DatabasePersistenceException;
+    public void addRoute (RouteEntity routeState) throws DatabasePersistenceException;
```
Comment (xieus) [inline services/route_manager/src/main/java/com/futurewei/alcor/route/service/RouteDatabaseService.java:15]: routeState => route

File: services/private_ip_manager/src/main/resources/application.properties [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 #spring.redis.host=localhost
 #spring.redis.port=6380
 #apache.kafka.address=172.17.0.1:9092
 #server.port=9000
 #Ignite configuration
 #ignite.host=192.168.137.1
-ignite.host=localhost
-ignite.port=10801
+ignite.host=192.168.137.1
+ignite.port=10800
```
Comment (xieus) [inline services/private_ip_manager/src/main/resources/application.properties:10]: I think this change caused UT failure.

File: kubernetes/services/subnet_manager.yaml [hunk 6]
```diff
@@ -4,17 +4,17 @@ metadata:
...
     logging.level.root=info
     logging.level.org.springframework.web=info
     logging.file.path=.
     logging.type=file
     apache.kafka.address=172.17.0.1:9092
-    microservices.vpc.service.url=http://192.168.1.25:30001/project/
-    microservices.mac.service.url=http://192.168.1.25:30005/macs
-    microservices.route.service.url=http://192.168.1.25:30003/
-    microservices.ip.service.url=http://192.168.1.25:30007/ips/
+    microservices.vpc.service.url=http://10.213.43.166:30001/project/
```
[the remark this answers is not in the document]
Comment (kevin-zhonghao) [reply kubernetes/services/subnet_manager.yaml:14]: Yes ~ when I set up another cluster, I will change it to multiple-node

File: services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java (hunk 2 above)
Comment (kevin-zhonghao) [reply services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java:53]: Sure

File: services/private_ip_manager/src/main/resources/application.properties (hunk 5 above)
Comment (kevin-zhonghao) [reply services/private_ip_manager/src/main/resources/application.properties:10]: Yeah you are right

File: kubernetes/services/subnet_manager.yaml (hunk 6 above)
[the remark this answers is not in the document]
Comment (xieus) [reply kubernetes/services/subnet_manager.yaml:14]: Okay thanks.

File: services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java [hunk 7]
```diff
@@ -50,7 +50,7 @@
             value = {"/vpcs/{vpcId}/routes/{routeId}"})
     public RouteWebJson getRuleByVpcId(@PathVariable String vpcId, @PathVariable String routeId) throws Exception {
 
-        RouteWebObject routeState = null;
+        Route routeState = null;
```
Comment (xieus) [inline services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java:53]: @kevin-zhonghao The change I was referring to was "RouteEntity route = null" :-)

Comment (kevin-zhonghao) [reply services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java:53]: Sorry ~ I will change it right now

File: services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java [hunk 8]
```diff
@@ -50,49 +50,49 @@
             value = {"/vpcs/{vpcId}/routes/{routeId}"})
     public RouteWebJson getRuleByVpcId(@PathVariable String vpcId, @PathVariable String routeId) throws Exception {
 
-        Route routeState = null;
+        RouteEntity routeEntity = null;
```
Comment (xieus) [inline services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java:53]: This is good.

File: services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/RouteDatabaseServiceImpl.java [hunk 9]
```diff
@@ -35,9 +35,9 @@ public Map getAllRoutes() throws CacheException {
     }
 
     @Override
-    public void addRoute(Route routeState) throws DatabasePersistenceException {
+    public void addRoute(RouteEntity routeEntityState) throws DatabasePersistenceException {
```
Comment (xieus) [inline services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/RouteDatabaseServiceImpl.java:38]: => "RouteEntity routeEntity"

File: services/route_manager/src/main/java/com/futurewei/alcor/route/service/RouteDatabaseService.java [hunk 10]
```diff
@@ -4,15 +4,15 @@
...
 
 import java.util.Map;
 
 public interface RouteDatabaseService {
 
-    public Route getByRouteId (String routeId) throws ResourceNotFoundException, ResourcePersistenceException;
+    public RouteEntity getByRouteId (String routeId) throws ResourceNotFoundException, ResourcePersistenceException;
     public Map getAllRoutes () throws CacheException;
-    public void addRoute (Route routeState) throws DatabasePersistenceException;
+    public void addRoute (RouteEntity routeEntityState) throws DatabasePersistenceException;
```
Comment (xieus) [inline services/route_manager/src/main/java/com/futurewei/alcor/route/service/RouteDatabaseService.java:15]: => "RouteEntity routeEntity"

File: services/route_manager/src/test/java/com/futurewei/alcor/route/RouteEntityControllerTests.java [hunk 11]
```diff
@@ -29,7 +29,7 @@
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
         properties = {"httpbin=http://localhost:${wiremock.server.port}"})
 @AutoConfigureMockMvc
-public class RouteControllerTests {
+public class RouteEntityControllerTests {
```
Comment (xieus) [inline services/route_manager/src/test/java/com/futurewei/alcor/route/RouteEntityControllerTests.java:32]: Revert back to "RouteControllerTests"

File: docs/modules/ROOT/pages/db_services/data_store.adoc [hunk 12]
```diff
@@ -53,14 +53,14 @@ Similar analysis could be done for other controllers.
 ** The target is 1 million read/write operations per second.
 ** Estimated size of a middle-sized region is 500,000 node with 100 ports per node with consideration of multi-port container and VMs.
 Thus, the estimated total number of ports is ~50 millions.
-** The estimated number of port-related resources (e.g. security groups, ACL, routes, DNS records and subnets) is of the same magnitude.
+** The estimated number of port-related resources (e.g. security groups, ACL, routeEntities, DNS records and subnets) is of the same magnitude.
```
Comment (xieus) [inline docs/modules/ROOT/pages/db_services/data_store.adoc:56]: I think we need to revert the changes in *.adoc, *.puml, *.proto and application.properties.

Global replacement is dangerous :-)

File: services/route_manager/src/main/java/com/futurewei/alcor/route/service/RouteDatabaseService.java (hunk 10 above)
Comment (kevin-zhonghao) [reply services/route_manager/src/main/java/com/futurewei/alcor/route/service/RouteDatabaseService.java:15]: Sure

File: docs/modules/ROOT/pages/db_services/data_store.adoc (hunk 12 above)
Comment (kevin-zhonghao) [reply docs/modules/ROOT/pages/db_services/data_store.adoc:56]: It's my bad ~ I will fix them

Comment (kevin-zhonghao) [reply docs/modules/ROOT/pages/db_services/data_store.adoc:56]: Yeah you are right ! it's my fault

Comment (xieus) [reply docs/modules/ROOT/pages/db_services/data_store.adoc]: It is okay. It happened. Don't worry about it. 

Waiting for the latest CI results to merge the PR.
